{{cookiecutter.repo_name}}/.pre-commit-config.yaml

```diff
@@ -48,18 +48,18 @@ repos:
         args: ['--ignore=E203,E266,E501,W503', '--max-line-length=88', '--max-complexity=18', '--select=B,C,E,F,W,T4']
 
 - repo: https://github.com/timothycrosley/isort
```

Comment:
Same as above, the offical repo appears to have been brought under the PyCQA umbrella https://github.com/PyCQA/isort, so perhaps we update the URL to match?